src/main/webapp/resources/js/pages/projects/settings/components/processing/ProcessingPriorities.jsx

```diff
@@ -12,8 +13,8 @@ import { putPriorityUpdate } from "../../../redux/projectSlice";
  * @constructor
  */
 export function ProcessingPriorities({ projectId }) {
-  const dispatch = useDispatch();
-  const { priority } = useSelector((state) => state.project);
+  const [updateProjectPriority] = useUpdateProjectPriorityMutation();
+  const { data: project } = useGetProjectDetailsQuery(projectId);
```

Comment:
Shouldn't this be 

`const { data: project = {} } = useGetProjectDetailsQuery(projectId);`

Reply:
It doesn;t need to be but it also doesn't hurt.  I updated it in 4d5694d.  By the time this gets called here, it is already called in one of the other components, so this is just getting the cached data therefore does not have the opportunity to call for an attribute on undefiend.

Reply:
Make sense. Thanks for updating it though :)